### ISSUE TITLE
Update dependences?

### ISSUE THREAD

**Issue by bnbon** [hapipal/haute-couture#86]
Hello, this is a great plugin, but is it maintained? Some of the dependencies fail with the latest Hapi

Thanks

**Comment by devinivy** [comment]
Yes!  hapi v21 was just published this week, and there's some work to do across the entire hapi pal ecosystem to bring it all into alignment.  I will be working on this v soon :)
